bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -43,6 +43,48 @@ __date__ = pycbc.version.date
 __program__ = "pycbc_pygrb_pp_workflow"
 
 
+# Function used to retrive a File instance from a FileList based on an
+# injection set label
+def fish_label_in_filelist(label, filelist):
+    """Return the File in a FileList with label in its name if the label
+    appears only once in the FileList, otherwise return None.
+    """
+    out_file = None
+    if filelist is not None:
+        out_file = list(filter(lambda x: label.lower()
+                        in x.name.lower(), filelist))
+        out_file = out_file[0] if len(out_file) == 1 else None
+
+    return out_file
+
+
+# Function to retrieve the segments plot (produced in the preprocessing stage)
+# that ensures its copy is saved in the output directory before returning to
+# the original working directory. Appropriate meta data is added to the plot.
+def display_seg_plot(output_dir, segments_dir):
```

Comment:
Do we need to use `glob`s here? If the segment file name is predictable, can we just explicitly look for it? (Bonus points if the name is passed through explicitly, either as a file object, or (if the preprocessing stage is a separate executable) as a path on the command line on in the ini file?